.classpath

```diff
@@ -9,6 +9,7 @@
 	</classpathentry>
```

Comment:
.classpath should be removed and ignored. Eclipse will generate it from the pom

Reply:
Not with hybrid projects unfortunately. And this is a Lua / Java hybrid.